pilot/pkg/features/pilot.go

```diff
@@ -157,6 +157,14 @@ var (
 		false,
 		"If enabled, controller that untaints nodes with cni pods ready will run. This should be enabled if you disabled ambient init containers.").Get()
 
+	ConnectOriginateIdleTimeout = env.Register(
+		"PILOT_CONNECT_ORIGINATE_IDLE_TIMEOUT",
```

Comment:
Can we name this something more descriptive/avoid exposing implementation details through the naming of environment variables.

Maybe something like `EnvoyHBONEIdleTimeout`.